affinity/cpp-20/d0796r3.md

```diff
@@ -468,25 +497,37 @@ The `execution_resource` class provides an abstraction over a system's hardware,
 
 *Returns:* The total concurrency available to this resource. More specifically, the number of *threads of execution* collectively available to this `execution_resource` and any resources which are *members of*, recursively.
 
-    std::vector<resource> resources() const noexcept;
+    size_type size() const noexcept;
+
+*Returns:* The number of child `execution_resource`s.
+
+    const_iterator begin() const noexcept;
 
-*Returns:* All `execution_resource`s which are *members of* this resource.
+*Returns:* A const iterator to the beggining of the child `execution_resource`s.
+
+    const_iterator end() const noexcept;
+
+*Returns:* A const iterator to the end of the child `execution_resource`s.
+
+    const_reference operator[](int child) const noexcept;
```

Comment:
shouldn't this be size_type child instead of int?

Reply:
Fixed.